dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/FilteredItemConverter.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.converter;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Logger;
+import org.dspace.app.rest.model.FilteredItemRest;
+import org.dspace.app.rest.model.MetadataValueList;
+import org.dspace.app.rest.projection.Projection;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataField;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.discovery.IndexableObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * This class "overrides" ItemConverter for the Filtered Items report. It handles FilteredItemRest
+ * instances, which add the owning collection for a given object.
+ *
+ * @author Jean-François Morin (Université Laval)
+ */
+@Component
+public class FilteredItemConverter
+        extends DSpaceObjectConverter<Item, FilteredItemRest>
+        implements IndexableObjectConverter<Item, FilteredItemRest> {
+
+    @Autowired
+    private ItemService itemService;
+    @Autowired
+    private CollectionConverter collectionConverter;
+
+    private static final Logger log = org.apache.logging.log4j.LogManager.getLogger(FilteredItemConverter.class);
+
+    @Override
+    public FilteredItemRest convert(Item obj, Projection projection) {
+        FilteredItemRest item = super.convert(obj, projection);
+        item.setInArchive(obj.isArchived());
+        item.setDiscoverable(obj.isDiscoverable());
+        item.setWithdrawn(obj.isWithdrawn());
+        item.setLastModified(obj.getLastModified());
+        // Here we handle the owning collection. The remainder of the class is identical to ItemConverter
+        // (as of DSpace 7.4).
```

Comment:
I would be delighted to merge my FilteredItemConverter-based additions into existing class ItemConverter. I think I remember having typing issues between ItemRest and FilteredItemRest, but I will have to check this again.

Another solution may consist in merging my parent collection attribute (from FilteredItemRest) into ItemRest. If you agree with this, I will do so and this will allow eliminating FilteredItemConverter as well. Personally, I think this will be the best way to proceed. Hopefully it will not decrease performance for other REST operations involving ItemRest instances created through ItemConverter.